### PR TITLE
When possible

### PR DESCRIPTION
**Pull request by SColchester**
Added a new subhashtag

### CODE REVIEW

File: app/_partner/factset.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ subhashtags:
   - factset20singapore
   - factset20sydney
   - factset20tokyo
+  - factsetgermany
```
Comment (dakotabenjamin) [inline app/_partner/factset.md:54]: should this be `factset20germany`?

Comment (SColchester) [reply app/_partner/factset.md:54]: Weirdly not, they had a mapathon today that mistakenly used this old hashtag and now they want to see how it compares

Comment (dakotabenjamin) [reply app/_partner/factset.md:54]: ok cool. merging now

Comment (SColchester) [reply app/_partner/factset.md:54]: Many thanks Dakota!

**Pull request merged**